modules/extensions/partials/proc_authenticating-with-openshift-connector-from-che.adoc

```diff
@@ -1,53 +1,63 @@
 // using-openshift-connector-in-eclipse-che
 
-
-
 [id="authenticating-with-openshift-connector-from-{prod-id-short}_{context}"]
-= Authenticating with OpenShift Connector from {prod-short}
+= Authenticating with OpenShift Connector from {prod-short} when the OpenShift OAuth service does not authenticate the {prod-short} instance
 
-The following section is relevant only when the OpenShift OAuth service does not already authenticate a {prod-short} instance; otherwise, the OpenShift Connector plug-in automatically establishes the authentication with the Openshift instance where {prod-short} runs.
+This section describes how to authenticate with an OpenShift cluster when the OpenShift OAuth service does not authenticate the {prod-short} instance. It enables the user to develop and push Components from {prod-short} to the OpenShift instance containing {prod-short}.
 
-Before the user can develop and push Components from {prod-short}, they need to authenticate with an OpenShift cluster.
+[NOTE]
+====
+When the OpenShift OAuth service does authenticate the {prod-short} instance, the OpenShift Connector plug-in automatically establishes the authentication with the OpenShift instance containing {prod-short}.
+====
 
 OpenShift Connector offers the following methods for logging in to the OpenShift Cluster from the {prod-short} instance:
 
-* Using the notification that asks to log in to the OpenShift cluster where {prod-short} is deployed to.
+* Using the notification asking to log in to the OpenShift instance containing {prod-short}.
```

Comment:
Not sure why we need gerunds here. IMO "that asks" is less ambiguous than the gerund.

Reply:
`that` triggers a SG error...

Reply:
Damn, ok, let's leave the gerunds then.